app/javascript/flavours/glitch/styles/components/index.scss

```diff
@@ -1,3 +1,29 @@
+@import 'boost';
+@import 'accounts';
+@import 'domains';
+@import 'status';
+@import 'modal';
+@import 'compose_form';
+@import 'columns';
+@import 'regeneration_indicator';
+@import 'directory';
+@import 'search';
+@import 'emoji';
+@import 'doodle';
+@import 'drawer';
+@import 'media';
+@import 'sensitive';
+@import 'lists';
+@import 'emoji_picker';
+@import 'local_settings';
+@import 'error_boundary';
+@import 'single_column';
+@import 'announcements';
+@import 'explore';
+@import 'signed_out';
+@import 'privacy_policy';
+@import 'about';
+
```

Comment:
This should be reverted and put back at the bottom.
The imports actually overwrite some styling set in this file.

To stop stylelint from complaining, add the following comment above the imports:
```scss
/* stylelint-disable no-invalid-position-at-import-rule -- Abiding to rule causes wrong styling */
```